JellyfinPlayer/VideoPlayerSettingsView.swift

```diff
@@ -80,6 +95,7 @@ struct VideoPlayerSettings: View {
         .onAppear(perform: {
             _captionTrack.wrappedValue = self.delegate.selectedCaptionTrack
             _audioTrack.wrappedValue = self.delegate.selectedAudioTrack
+            playbackSpeedSelection = self.delegate.selectedPlaybackSpeedIndex
```

Comment:
Is there any advantage to assigning the value with `_object.wrappedValue`  compared to just assigning the variable directly?

Reply:
No and I prefer just assigning it directly. This is the first time I've used Swift and didn't know that both worked. If you see any, feel free to change them.